pkg/controller/tidbcluster/tidb_cluster_controller.go

```diff
@@ -139,6 +139,7 @@ func NewController(
 				pdFailover,
 			),
 			mm.NewTiKVMemberManager(
+				cmInformer.Lister(),
```

Comment:
I recall that this would cause tidb operator to use much more memory especially for large clusters, e.g. our IDC. @aylei